apps/aehttp/test/aehttp_sc_SUITE.erl

```diff
@@ -530,8 +597,10 @@ sc_ws_open_(Config, ChannelOpts0, MinBlocksToMine) ->
                  get_accounts_by_pubkey_sut(aeser_api_encoder:encode(account_pubkey, IPubkey)),
     {ok, 200, #{<<"balance">> := RStartAmt}} =
                  get_accounts_by_pubkey_sut(aeser_api_encoder:encode(account_pubkey, RPubkey)),
-    IAmt = 70000 * aec_test_utils:min_gas_price(),
-    RAmt = 40000 * aec_test_utils:min_gas_price(),
+    IAmt = maps:get(initiator_amount, ChannelOpts0,
```

Comment:
If provided, use the `initiator_amount` and `responder_amount` for opening amounts. This is really important if later on we expect the channel to have enough tokens to pay for the fat close mutual fee :)